mod_reforged/hooks/skills/actives/taunt.nut

```diff
@@ -28,21 +28,46 @@
 
 	q.onUse = @(__original) function( _user, _targetTile )
 	{
-		__original(_user, _targetTile);
+		local ret = __original(_user, _targetTile);
+		if (!ret) return ret;
 
 		local target = _targetTile.getEntity();
-		if (this.getContainer().getActor().getTile().getDistanceTo(_targetTile) <= this.m.MaxRangeDefenseDebuff)
+		if (this.getContainer().getActor().getTile().getDistanceTo(_targetTile) <= this.m.MaxRangeForDefenseDebuff)
 		{
 			local tauntEffect = target.getSkills().getSkillByID("effects.taunted");
 			tauntEffect.m.DefenseModifier = this.calculateDefenseModifier();
 		}
 		target.getSkills().update();	// Otherwise the defense debuff will not be immediately visible in the combat tooltip
+
+		return true;
+	}
+
+// MSU Functions
+	q.getNestedTooltip = @(__original) function()
```

Comment:
Should be `@()` because you aren't using the `__original()` in the new function definition.